ami/flowchart/library/Roi.py

```diff
@@ -263,9 +271,18 @@ def to_operation(self, **kwargs):
         ex = self.values['extent x']
         oy = self.values['origin y']
         ey = self.values['extent y']
+        if hasattr(self, 'rotation'):
+            rotate = self.rotation
+        else:
+            rotate = 0
+
+        roi = self.roi
+        image_item = self.widget.imageItem
 
         def func(img):
-            return img[slice(ox, ox+ex), slice(oy, oy+ey)], (ox, ex, oy, ey)
+            #print(rotate)
+            return np.rot90(img.T, rotate)[slice(ox, ox+ex), slice(oy, oy+ey)], (ox, ex, oy, ey)
+            #return roi.getArrayRegion(image_item.image, image_item)
```

Comment:
@ddamiani  This would be the way to do it, but the Widgets are not pickable. I am not sure there is a good workaround that?